### PR TITLE
Replace webapp alerts from jsonnet to YAML

### PR DESCRIPTION
**Pull request by ArthurSens**
Signed-off-by: ArthurSens <arthursens2005@gmail.com>

## Description

As socialized a couple of times before, our colleagues at Gitpod do not work well with Jsonnet, so we're slowly getting rid of all code we have in this language. 

With the new observability installer, we're able to import YAML files stored in git. This means that we can replace alerts written in jsonnet with raw YAML that represents the PrometheusRule manifests we need.

## Related Issue(s)

Fixes #

## Release Notes

```release-note
NONE
```

## Werft options:

- [ ] /werft with-preview

### CODE REVIEW

File: .werft/observability/monitoring-satellite.ts
Comment (ArthurSens) [inline .werft/observability/monitoring-satellite.ts:121]: These same lines will need to be added to all monitoring-satellite apps that we have in argocd

**Comment by ArthurSens** [comment]
/hold 

because merge needs to be coordinated with changes made to ArgoCD apps

**Review by geropl — APPROVED** [review]
Thank you @ArthurSens ! :pray: 

:heavy_check_mark: to unblock

**Review by vulkoingim — APPROVED** [review]

**Pull request merged**
